### PR TITLE
Document dilemma customization

### PR DESCRIPTION
**Pull request by bstiq**

### CODE REVIEW

File: fw/dilemma-features.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+DPI (i.e. dots per linear inch), a.k.a. mouse sensitivity, can be controlled by the firmware. The Dilemma keymap offers 2 different DPI settings:
+
+- **Default** DPI: the sensitivity of the pointer in normal mode.
+- **Sniping** DPI: the sensitivity of the pointer in [sniping mode](#sniping)
+
+For each mode, the firmware allows cycling through multiple pre-defined values.
+
+- Default mode:
+    - Default value: 400 DPI
```
Comment (burkfers) [inline fw/dilemma-features.md]: Dilemma starts at step 3 (400+3x200) iirc - double check

File: fw/dilemma-features.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+### Drag-scroll
+
+**Drag-scroll** enables scrolling with the trackpad. When drag-scroll is enabled, the trackpad's `x` and `y` movements are converted into `h` (horizontal) and `v` (vertical) movement, effectively sending scroll instructions to the host system.
+
+Custom keycodes:
+
+| Name   | Description                                           |
+| ------ | ----------------------------------------------------- |
+| `DRGSCRL`  | enable drag-scroll mode as long as the key is pressed |
```
Comment (burkfers) [inline fw/dilemma-features.md:143]: Could optionally mention the long name aliases

Comment (bstiq) [reply fw/dilemma-features.md:143]: iirc the long ones don't work in VIA, so let's keep it simple

File: fw/dilemma-features.md (hunk 1 above)
Comment (bstiq) [reply fw/dilemma-features.md]: good catch, thanks!

**Pull request merged**